examples/debian-cli.ini

```diff
@@ -0,0 +1,5 @@
+# If you're using Debian/Ubuntu, put this file into
+# /etc/letsencrypt/cli.ini
```

Comment:
While config files are slightly broken now (#640), it should not affect Debian package, which doesn't use `master`, as we pretty much do in `requirements.txt`. "Normal" users cannot use the advice given above, at least for now.